server/frontend/dist/index.html

```diff
@@ -9,6 +9,7 @@
 
   <body>
     <h1>Shotgrid Sync</h1>
+    <button id="sync-users-button"  class="sync-users-button" onclick="syncUsers();">Sync Users</button>
```

Comment:
- It would be beneficial to include details about the direction of sync.
- Additionally, what are the implications if a user has already been synced and subsequently leaves ShotGrid?
- Will they be removed from AY's database or simply archived?

Reply:
We don't delete users in SG, we simply mark them unactive, I could also add functionality to sync unactive users to keep both on sync... but we have so many unactive users I'd rather not do that. Ideally that would happen through the event system instead of a button so if an Ayon user gets marked unactive it gets propagated to SG and viceversa

Similarly as I mentioned in Discord yesterday, the next step is to add support for the users access/roles to grab that info from SG. Studios that have SG won't be doing the user access from Ayon, they will simply do it from SG so we should catch that event in the project entity and add the users list to the Ayon project